src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/Loader/PeopleTable';
+import { Person } from '../types';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [showError, setShowError] = useState('');
+
+  const errorMessage = (message: string) => {
+    setShowError(message);
+  };
+
+  const getPeopleFromServer = async () => {
+    try {
+      const dataFromServer = await getPeople();
+
+      const peopleWithParents = dataFromServer.map((person: Person) => {
+        return {
+          ...person,
+          mother: dataFromServer.find(
+            (p: Person) => p.name === person.motherName,
+          ),
+          father: dataFromServer.find(
+            (p: Person) => p.name === person.fatherName,
+          ),
+        };
+      });
+
+      setPeople(peopleWithParents);
+    } catch {
+      errorMessage('Something went wrong');
+    }
+  };
+
+  useEffect(() => {
+    getPeopleFromServer();
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+      <div className="block">
+        <div className="box table-container">
+          {people.length === 0 && (
```

Comment:
What if the server returns an empty array? We will see an infinite loader.
Let's create a separate state for loading logic